docs/0005-hashed-n-tuple-id-layout.md

```diff
@@ -0,0 +1,235 @@
+# OCFL Community Extension 0005: Hashed Truncated N-tuple Trees with Encapsulating Directory for OCFL Storage Hierarchies
+
+  * **Extension ID:** 0005-hashed-n-tuple-id-layout
```

Comment:
I need to update my PR too, but it sounds like the language in the spec is in the process of being updated to call this "Extension Name".

Reply:
fixed